docs-src/clusters/how-to-migrate-visibility-database.md

```diff
@@ -79,6 +112,8 @@ The following steps describe how to migrate your Visibility database with exampl
            scheme: "http"
            host: "127.0.0.1:9200"
            indices:
-           visibility: temporal_visibility_v1
+           visibility: temporal_visibility_v1_dev
            closeIdleConnectionsInterval: 15s
    ```
+
+You can also remove the secondary Visibility dynamic configuration in your self-hosted Cluster configuration.
```

Comment:
I think you meant this, right?

```suggestion
You can also remove the secondary Visibility configuration in your self-hosted Cluster configuration.
```

Reply:
I meant to call out that you can remove this bit from the dynamic configuration yaml

```yaml
   system.secondaryVisibilityWritingMode:
   - value: "dual"
   constraints: {}
   system.enableReadFromSecondaryVisibility:
   - value: true
   constraints: {}
   ```
I don't think it affects the cluster (but please confirm) but is probably a good idea to remove unnecessary config details

Reply:
But you already said to remove the dynamic config in the previous paragraph:

> (...), and remove the dynamic configuration set in the previous steps.

Reply:
yeah you're right' removed the repetitive statement.